### PR TITLE
Support nested objects

### PR DESCRIPTION
**Pull request by benjreinhart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,11 +31,36 @@ Example:
 * `entryPoint` is a file relative to `process.cwd()` that will be the initial module marked for inclusion in the bundle as well as the exported module
 * `root` is the directory to which unqualified requires are relative; defaults to `process.cwd()`
 * `options` is an optional object (defaulting to `{}`) with zero or more of the following properties
-    * `export`: a variable name to add to the global scope; assigned the exported object from the `entryPoint` module
+    * `export`: a variable name to add to the global scope; assigned the exported object from the `entryPoint` module. This option can be an object with any level of nesting, i.e. `App.Models.User`. Note that, in this example, `App` would have to be previously defined on the global object and its accessed properties would also have to be defined otherwise a type error will be thrown.
```
Comment (michaelficarra) [inline README.md]: "Any valid [Left-Hand-Side Expression](http://es5.github.com/#x11.2) may be given instead." is sufficient.

File: src/index.coffee [hunk 2]
```diff
@@ -164,11 +164,7 @@ exports.cjsify = (entryPoint, root = process.cwd(), options = {}) ->
       expression:
         type: 'AssignmentExpression'
         operator: '='
-        left:
-          type: 'MemberExpression'
-          computed: true
-          object: { type: 'Identifier', name: 'global' }
-          property: { type: 'Literal', value: options.export }
+        left: buildExportExpression options.export.split /\./
```
Comment (michaelficarra) [inline src/index.coffee]: No. No no no.

``` coffee
exportExpression = (esprima.parse options.export).body[0].expression
lhs =
  if exportExpression.type is 'Identifier'
    type: 'MemberExpression'
      computed: false
      object: { type: 'Identifier', name: 'global' }
      property: { type: 'Identifier', value: exportExpression.name }
  else
    exportExpression
```

File: README.md [hunk 3]
```diff
@@ -31,11 +31,36 @@ Example:
 * `entryPoint` is a file relative to `process.cwd()` that will be the initial module marked for inclusion in the bundle as well as the exported module
 * `root` is the directory to which unqualified requires are relative; defaults to `process.cwd()`
 * `options` is an optional object (defaulting to `{}`) with zero or more of the following properties
-    * `export`: a variable name to add to the global scope; assigned the exported object from the `entryPoint` module
+    * `export`: Any valid [Left-Hand-Side Expression](http://es5.github.com/#x11.2) may be given instead.
```
Comment (michaelficarra) [inline README.md]: You were supposed to append, not replace.

Comment (benjreinhart) [inline README.md]: you said this was sufficient. Your mistake.

Comment (michaelficarra) [inline README.md]: A sufficient addition, instead of what you wrote.

**Pull request merged**